src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/ResourceUrlFilter.cs

```diff
@@ -35,10 +35,13 @@ public ValueTask<FluidValue> ProcessAsync(FluidValue input, FilterArguments argu
                 resourcePath = _httpContextAccessor.HttpContext.Request.PathBase.Add(resourcePath.Substring(1)).Value;
             }
 
-            // Don't prefix cdn if the path is absolute, or is in debug mode.
+            // Don't prefix cdn if the path includes a protocol, i.e. is an external url, or is in debug mode.
             if (!_options.DebugMode
                 && !String.IsNullOrEmpty(_options.CdnBaseUrl)
-                && !Uri.TryCreate(resourcePath, UriKind.Absolute, out var uri))
+                // Don't evaluate with Uri.TryCreate as it produces incorrect results on Linux.
+                && !resourcePath.StartsWith("https://", StringComparison.OrdinalIgnoreCase)
```

Comment:
Formatting: in the aspnetcore repo they put the `&&` at the end of the upper line

But there is no universal rule ;)

Reply:
Done. I tend to the other for easier readability, but that's just me ;)